atlasdb-workload-server/src/main/java/com/palantir/atlasdb/workload/workflow/RandomWorkflows.java

```diff
@@ -111,8 +117,37 @@ private List<DeleteTransactionAction> generateDeleteActions() {
                     .collect(Collectors.toList());
         }
 
+        private List<RowColumnRangeReadTransactionAction> generateRowColumnRangeReadActions() {
+            return IntStream.range(0, random.nextInt(workflowConfiguration.maxRowRangeScans() + 1))
```

Comment:
no action: This is fun - two separate exclusive endpoints (next int goes to X+1 -> max is X, then range goes up to X - 1 leaving X values). Nothing required, was just fun to double check that it was correct.

Reply:
Yeah - in the original PR there was a bit of funky discussion around this as well :)

Reply:
basically, annoyingly you need to pick from N+1 outcomes, and given the APIs this drops out